libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html

```diff
@@ -0,0 +1,24 @@
+<al-card alGradientCard>
+  <div alCardContent>
+    <div class="flex flex-col md:flex-row lg:flex-col">
```

Comment:
Hmmm maybe container queries is better to use in this case ?
see https://github.com/tailwindlabs/tailwindcss-container-queries

Reply:
I hadn't known container queries. Both solutions works properly but I decided to use tailwind container queries. Thank you